### PR TITLE
fix(inline-input): fix to input width that affects input icon - FE-2671

### PR DESCRIPTION
**Pull request by damianOfKyan**
Fixes FE-2671
Fixes GitHub Issue: #2748

### Proposed behaviour
InlineInputs should display a validation icon correctly by default aligned to the right hand side of the input field.

inline-input-fixed

### Current behaviour
InlineInputs are using up 100% of the width pushing a validation icon to the new line. (wrap)

Screenshot 2020-04-16 at 13 40 02

### Checklist

- [x] Release notes (Conventional Commits) 
- [x] Unit tests
<del>- [ ] Cypress automation tests
- [x] Storybook added or updated
- [x] Theme support
<del>- [ ] Typescript `d.ts` file added or updated

### Testing instructions
1. checkout branch
2. navigate to http://localhost:9001/?path=/story/inlineinputs--validations
3. Investigate first inline input that should have a icon to the right hand side

Inspecting element in the browser and changing input width: auto; to width: 100% will cause the issue to re-appear confirming that width: auto; resoles the issue.

### CODE REVIEW

File: src/components/inline-inputs/inline-inputs.spec.js [hunk 1]
```diff
@@ -67,9 +67,9 @@ describe('Inline Inputs', () => {
       }, wrapper, { modifier: '.carbon-row' });
     });
 
-    it('then all inputs should have 100% width', () => {
+    it('then all inputs should have auto width', () => {
```
Comment (grabkowski) [inline src/components/inline-inputs/inline-inputs.spec.js:70]: I think it's no longer `auto`

Comment (damianOfKyan) [reply src/components/inline-inputs/inline-inputs.spec.js]: fixed.

**Comment by sianford** [comment]
Tested in Safari, Chrome, Firefox and Edge (for Mac) browsers. All issues resolved. The validation icon no longer wraps. All components work correctly. Testing passed ✅

**Pull request merged**